frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -504,42 +410,38 @@ const ManagePolicyPage = (managePoliciesPageProps: {
           </InfoBanner>
         )}
         <div>
-          {!!selectedTeamId &&
-            (isTeamPoliciesError ? (
-              <TableDataError />
-            ) : (
-              <PoliciesListWrapper
-                policiesList={teamPolicies}
-                isLoading={
-                  isLoadingTeamPolicies && isLoadingFailingPoliciesWebhook
-                }
-                onRemovePoliciesClick={onRemovePoliciesClick}
-                canAddOrRemovePolicy={canAddOrRemovePolicy(
-                  currentUser,
-                  selectedTeamId
-                )}
-                selectedTeamData={selectedTeamData}
-                currentAutomatedPolicies={currentAutomatedPolicies}
-              />
-            ))}
-          {!selectedTeamId &&
-            (isGlobalPoliciesError ? (
-              <TableDataError />
-            ) : (
-              <PoliciesListWrapper
-                policiesList={globalPolicies}
-                isLoading={
-                  isLoadingGlobalPolicies && isLoadingFailingPoliciesWebhook
-                }
-                onRemovePoliciesClick={onRemovePoliciesClick}
-                canAddOrRemovePolicy={canAddOrRemovePolicy(
-                  currentUser,
-                  selectedTeamId
-                )}
-                selectedTeamData={selectedTeamData}
-                currentAutomatedPolicies={currentAutomatedPolicies}
-              />
-            ))}
+          {!!teamId && teamPoliciesError && <TableDataError />}
+          {!!teamId && !teamPoliciesError && isLoadingTeamPolicies && (
+            <Spinner />
+          )}
+          {!!teamId && !teamPoliciesError && !isLoadingTeamPolicies && (
```

Comment:
This would read a bit easier as a ternary operator, which is how we're handling the the spinner display elsewhere.